schema/deploy/tables/project_revision_004_add_pending_actions_from.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy cif:tables/project_revision_004_add_pending_actions_from to pg
```

Comment:
I'm a little confused about the AC for persisting the pending actions. The selection didn't survive a refresh when I tested it, and there's talk about an update button in the card comments